python/dgl/distributed/sampling.py

```diff
@@ -12,10 +13,10 @@
 __all__ = ['sample_neighbors']
 
 SAMPLING_SERVICE_ID = 6657
+INSUBGRAPH_SERVICE_ID = 6658
 
-
-class SamplingResponse(Response):
-    """Sampling Response"""
+class SubgraphResponse(Response):
```

Comment:
Why is this under sampling.py? Seems we need a new name for this file or move it to a separate one

Reply:
the code of `in_subgraph` has a large overlap with sampling. we could rename the file. what is a better name?